### DISCUSSION TITLE
教育與科技

### DISCUSSION THREAD

**Discussion by xipeiguo** [1101-GS4719-C/Python-Community-GS4719-C#2086] [General]
https://flipedu.parenting.com.tw/article/009519
在現今科技發達的時代，常常人手一台3C產品，而學校也是，在我的學習歷程中，從國中開始在課堂中漸漸有了平板的身影，每位同學都能分到一台，使我們能在課堂中活用，科技發展的助益下，讓我們能有更便利和在更有幫助的方式中學習，我想這對學生的幫助不容小覷。

**Comment by peresonacifer** [comment]
為了跟上時代的變化，現在的小朋友們要學的東西是真的挺多的，尤其是AI相關的科技，但是一定教導正確的使用方式，或是有方法判斷學生是否使用AI生成，因為作業和報告還是需要自己做，AI對學生來說是輔助工具，像我們高中英文老師都會叫我們用ChatGPT改英文作文。
